site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -64,17 +64,16 @@ The following table lists the currently available permissions:
     <tr id="background">
       <td><code>"background"</code></td>
       <td>
-        <p id="bg">Makes Chrome start up early and and shut down late, so that apps and extensions can have a longer
+        <p id="bg">Makes Chrome start up early and and shut down late, so that extensions can have a longer
           life.</p>
-        <p>When any installed hosted app, packaged app, or extension has "background" permission, Chrome runs
+        <p>When any installed extension has "background" permission, Chrome runs
           (invisibly) as soon as the user logs into their computer—before the user launches Chrome. The "background"
           permission also makes Chrome continue running (even after its last window is closed) until the user explicitly
           quits Chrome.</p>
-        <div class="aside aside--note"><b>Note:</b> Disabled apps and extensions are treated as if they aren't
+        <div class="aside aside--note"><b>Note:</b> Disabled extensions are treated as if they aren't
           installed.</div>
         <p>You typically use the "background" permission with a <a href="/docs/extensions/mv3/background_pages">background page</a>, <a
-            href="/docs/apps/event_pages">event page</a> or (for hosted apps) a <a
-            href="http://developers.google.com/chrome/apps/docs/background.html">background window</a>.</p>
+            href="/docs/apps/event_pages">event page</a>.</p>
```

Comment:
This now reads "use the 'background' permission with a background page, event page."

We should probably just have this be "use the 'background' permission with background scripts." (and link background scripts to https://developer.chrome.com/docs/extensions/mv3/background_pages/)